### ISSUE TITLE
[VL] Enable map_from_arrays function

### ISSUE THREAD

**Issue by WangGuangxin** [apache/gluten#6101]
### Description

Enable map_from_arrays function

**Comment by ArnavBalyan** [comment]
Hi @WangGuangxin I came across this, we need this support, just wanted to know if this is active. I can help cover any work that is left. Thanks

**Comment by WangGuangxin** [comment]
> Hi [@WangGuangxin](https://github.com/WangGuangxin) I came across this, we need this support, just wanted to know if this is active. I can help cover any work that is left. Thanks

@ArnavBalyan yes, its pending the pr https://github.com/facebookincubator/velox/pull/10280, you can help test and review this if you are convenient
